upload-perf/src/upload-perf.rs

```diff
@@ -74,7 +74,6 @@ fn main() {
                         ("commit", git_commit_hash.trim().to_string(), String)
                     );
                     */
-                    
```

Comment:
yeah, i also know this rustfmt differences between master,v1.6 rust tool versions... xD